code/game/gamemodes/revolution/revolution.dm

```diff
@@ -449,3 +456,49 @@
 	dat += "<HR>"
 
 	return dat
+
+
+/proc/revconvert(mob/living/user)
+	var/list/Possible = list()
+	for(var/mob/living/carbon/human/P in oview(user))
+		if((P.stat == CONSCIOUS) && P.client && P.mind && !P.mind.special_role)
+			Possible += P
+	if(!Possible.len)
+		user << "\red There doesn't appear to be anyone available for you to convert here."
```

Comment:
Any reason you're using the `<<` format instead of  `to_chat()`s in this proc?

Reply:
Theres also a byond `\red` text macro here. These were phased out when goonchat game, and should actually be banned by the travis configuration.

Reply:
Fixed, was using old shitcode and hadn't done the cleanup on it yet.